recipes/resolvo-cpp/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+cmake ${CMAKE_ARGS} -GNinja $SRC_DIR -DRust_CARGO_TARGET=${CARGO_BUILD_TARGET}
```

Comment:
Should `Release` be used as the build type?

```suggestion
cmake \
      -GNinja $SRC_DIR \
      -DRust_CARGO_TARGET=${CARGO_BUILD_TARGET} \
      -DCMAKE_BUILD_TYPE=Release \
      ${CMAKE_ARGS}
```

Reply:
I am pretty sure this is already included in the `CMAKE_ARGS`